src/assets/locales/fr.json

```diff
@@ -53,13 +74,17 @@
   "button.cancel": "Annuler",
   "button.continue": "Continuer",
   "button.next": "Suivant",
+  "chat-message-input.warning-max-characters": "Caractères max",
   "chat-sidebar-container.input-placeholder.empty-room": "Il n'y a personne pour le moment...",
   "chat-sidebar-container.input-placeholder.empty-room-bot": "Envoyer le message vers {discordChannels}",
   "chat-sidebar-container.input-placeholder.occupants": "{occupantCount, plural, one {Envoyer un message à un autre...} other {Envoyer un message aux {occupantCount} autres...} }",
   "chat-sidebar-container.input-placeholder.occupants-and-bot": "{occupantCount, plural, one {Envoyer un message à un autre et {discordChannels}...} other {Envoyer un message aux {occupantCount} autres et {discordChannels}...} }",
   "chat-sidebar.input.placeholder": "Message...",
   "chat-sidebar.log-message.audio-normalization-disabled": "l'audioNormalization est désactivée.",
   "chat-sidebar.log-message.audio-normalization-nan": "la commande audioNormalization nécessite un nombre valide dans les paramètres...",
+  "chat-sidebar.log-message.audio-resumed": "L'audio a repris.",
+  "chat-sidebar.log-message.audio-suspended": "L'audio a été suspendu, cliquez quelque part dans la salle pour reprendre l'audio.",
+  "chat-sidebar.log-message.avatar-changed": "Votre avatar a été modifié.",
```

Comment:
"chat-sidebar.log-message.audio-resumed": "Audio has been resumed.",
  "chat-sidebar.log-message.audio-suspended": "Audio has been suspended, click somewhere in the room to resume the audio.",
  "chat-sidebar.log-message.avatar-changed": "Your avatar has been changed.",

Reply:
Suggestion : "L'audio a été mis en pause, ..."